test/helpers/apollo-client.ts

```diff
@@ -86,7 +87,7 @@ export const createApolloClient = (
   }
 }
 
-export const getSubscriptionNext = (subscription: Observable<any>): Promise<any> => {
+export const getSubscriptionNext = (subscription: Observable<FetchResult<any>>): Promise<FetchResult<any>> => {
```

Comment:
any is a forbidden keyword per our typescript policy (we have something like `no-implicit-any: error` in our typescript conf). is that not possible to have it as `unknown`?

Reply:
I will try switching it to unknown